libs/agno/agno/db/postgres/postgres.py

```diff
@@ -1143,13 +1170,221 @@ def get_metrics_table(self) -> Table:
             )
         return self.metrics_table
 
-    def get_metrics(self) -> List[AggregatedMetrics]:
-        """Get all metrics from the database."""
-        return []
+    def get_metrics_raw(
+        self, starting_date: Optional[str] = None, ending_date: Optional[str] = None, table: Optional[Table] = None
```

Comment:
should the dates be str or int?

Reply:
Why do we pass in `Table` here? I see this elsewhere as well

Reply:
- Adjusted naming and typing for all the date fields
- `table` is useless here, will remove. The other raw methods can be called directly (API flow) or by the general methods (SDK flow), and when called by the general ones they receive the table, that's why it's a param in the other ones.